src/usb_moded-config.h

```diff
@@ -85,3 +86,8 @@ int is_roaming_not_allowed(void);
 
 int conf_file_merge(void);
 int set_config_setting(const char *entry, const char *key, const char *value);
+
+#define SET_CONFIG_ERROR (-1)
```

Comment:
These could be `enum` values?

```
enum set_config_result_t {
    SET_CONFIG_ERROR = -1,
    SET_CONFIG_UPDATED = 0,
    SET_CONFIG_UNCHANGED = 1,
};
```

Then you could also use `set_config_result_t` as the return type for all functions that return those values.